{"name": "Удаление курьера, по id не существующего курьера, не успешно", "status": "passed", "start": 1712509836070, "stop": 1712509836658, "uuid": "1c41fd30-f754-4af2-8bb3-73853f2cc759", "historyId": "80a5525c0b37e3738137c5911e8a89db", "testCaseId": "80a5525c0b37e3738137c5911e8a89db", "fullName": "tests.couriers.test_delete_courier.TestDeleteCourier#test_delete_courier_with_not_existing_courier_id_fail", "labels": [{"name": "parentSuite", "value": "tests.couriers"}, {"name": "suite", "value": "test_delete_courier"}, {"name": "subSuite", "value": "TestDeleteCourier"}, {"name": "host", "value": "DESKTOP-HCED1QB"}, {"name": "thread", "value": "6196-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_delete_courier"}]}